packages/cli/src/commands/releases/retry.ts

```diff
@@ -0,0 +1,54 @@
+import {Command, flags} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import color from '@heroku-cli/color'
+import * as Heroku from '@heroku-cli/schema'
+import {stream} from '../../lib/releases/output'
+import {findByLatestOrId} from '../../lib/releases/releases'
+
+export default class Retry extends Command {
+  static topic = 'releases';
+  static description = 'retry the latest release-phase command';
+  static examples = ['heroku releases:retry --app happy-samurai-42']
+  static help = 'Copies the latest release into a new one, retrying the latest release-phase command. App must have release-phase command.'
+  static flags = {
+    app: flags.app({required: true}),
+  };
+
+  public async run(): Promise<void> {
+    const {flags} = await this.parse(Retry)
+    const {app} = flags
+    const release = await findByLatestOrId(this.heroku, app)
+
+    if (!release) {
+      return ux.error('No release found for this app')
+    }
+
+    if (!release.slug) {
+      return ux.error('This command only works for apps using a release-phase command')
+    }
+
+    ux.action.start(`Retrying ${color.green('v' + release.version)} on ${color.app(app)}`)
+
+    const {body: retry} = await this.heroku.post<Heroku.Release>(`/apps/${app}/releases`, {
+      body: {
+        slug: release?.slug?.id,
+        description: `Retry of v${release.version}: ${release.description}`,
+      },
+    })
+
+    ux.action.stop(`done, ${color.green('v' + retry.version)}`)
+
+    if (retry.output_stream_url) {
+      ux.log('Running release command...')
+      await stream(retry.output_stream_url)
+        .catch(error => {
+          if (error.statusCode === 404 || error.response?.statusCode === 404) {
+            ux.warn('Release command starting. Use `heroku releases:output` to view the log.')
```

Comment:
Minor comment here. Can we modify this slightly to both use our `color.cmd` styling and add the name of the app as well? So like:
```javascript
ux.warn(`Release command starting. Use ${color.cmd('heroku releases:output --app ' + app)} to view the log.`)
```

or something.